doc/whatsnew.rst

```diff
@@ -4,13 +4,14 @@ What's new
 Next release
 ============
 
-Changes to :doc:`/api/tools-costs` (:pull:`186`, :pull:`187`)
+Changes to :doc:`/api/tools-costs` (:pull:`186`, :pull:`187`, :pull:`190`)
 -------------------------------------------------------------
```

Comment:
Please expand to cover the whole line above.
Alternatively, you can collect the `:pull:` statements down below (e.g. after every bullet point) to avoid the need of adapting these two lines constantly.